testutil/timestamp.go

```diff
@@ -0,0 +1,13 @@
+package testutil
```

Comment:
Please add license header

Reply:
You could copy it from other source file

Reply:
done